src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
Are we going to keep the same structure that we had before? I mean, are we still composing these sentences dynamically or are these new complete sentences? I'm not sure because you have used `activarán/desactivarán` instead of `activar\desactivar`.

Neither 1 or 2 are completely correct because of the tense that you're using. If you create the sentence like this you should have to use `activar/desactivar`. In case you want to use the future you will have to get rid of `van a`.

The order of the adjective has a very subtle difference... I would use `2.` But depending on whether we're composing this sentence dynamically or not maybe we could improve it.